README.md

```diff
@@ -50,3 +51,26 @@ Here a short description on how it is done:
 * for GNU Social: It can get the user from the URL, if the URL is `/main/ostatus/nickname/<username>`.
 * for Pleroma: If `/main/ostatus` is accessed, it tries to get the the username from the referrer. If not possible, it tries to scrape it from the website.
 * for Friendica: It can get the user from the URL, if the URL is `/main/dfrn_request/<username>`.
+
+## Contributors ✨
```

Comment:
one place is enough, please remove it from the Readme :thinking:

Reply:
Sure, I'll remove it.